src/components/Main/Main.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoItem } from '../TodoItem';
+
+type Props = {
+  todos: Todo[],
+  tempTodo: Todo | null,
+  isTempLoading: boolean,
+  setTodos: (todos: Todo[]) => void,
+  pushError: (title: string) => void,
+  toBeCleared: Todo[] | undefined,
```

Comment:
Consider initializing 'toBeCleared' as an empty array instead of 'undefined' to simplify the code.